src/series.py

```diff
@@ -0,0 +1,44 @@
+"""This module calculates Fibonacci and Lucas sequences."""
+
+
+def fibonacci(n):
+    """Return the nth number of a Fibonacci sequence."""
+    series = []
+    for i in range(n):
+        if i == 0 or i == 1:
+            series.append(i)
+        else:
+            next_num = (series[i - 2]) + (series[i - 1])
+            series.append(next_num)
+    return series[-1]
+
+
+def lucas(n):
+    """Return the nth number of a Lucas sequence."""
+    series = []
+    for i in range(n):
+        if i == 0:
+            series.append(2)
+        elif i == 1:
+            series.append(1)
+        else:
+            series.append(series[i - 2] + series[i - 1])
+    return series[-1]
+
+
```

Comment:
There is a lot of repeated code in this program, which is bad. Remember DRY -- Don't Repeat Yourself. Repeated code is burdensome and hard to maintain. Since `fibonacci` and `lucas` are just more specific versions of `sum_series`, Can you think of how you could re-use code to prevent any repetitions?